### PR TITLE
fix: update video.js, contrib-dash, and chromecast

### PR DESCRIPTION
**Pull request by gkatsev**

### CODE REVIEW

File: videojs-contrib-dash.html [hunk 1]
```diff
@@ -17,19 +17,14 @@
...
-  <script src="//cdn.dashjs.org/latest/dash.all.debug.js"></script>
-  <script src="https://npmcdn.com/videojs-contrib-dash@^2.6.1/dist/videojs-dash.js"></script>
-  <link href="//vjs.zencdn.net/5.10/video-js.css" rel="stylesheet">
+  <!-- Video.js, DashJS -->
+  <script src="//vjs.zencdn.net/7.16.0/video.js"></script>
+  <script src="https://unpkg.com/videojs-contrib-dash@5.1.0/dist/videojs-dash.js"></script>
+  <link href="//vjs.zencdn.net/7.16.0/video-js.css" rel="stylesheet">
 
   <!-- Chromecast post-load -->
-  <script src="js/videojs-chromecast/videojs-chromecast.min.js"></script>
```
Comment (philcluff) [inline videojs-contrib-dash.html:31]: Can you `rm` these files from the code base too as it's being replaced with the CDN version?

**Review by philcluff — COMMENTED** [review]
Just one minor change to make to clean up unused files :)

File: videojs-contrib-dash.html (hunk 1 above)
Comment (gkatsev) [reply videojs-contrib-dash.html:31]: this file is still used in the main cast file. But I can switch it out to the CDN version too, I guess and then remove the local files.

Comment (gkatsev) [reply videojs-contrib-dash.html:31]: updated

**Review by philcluff — APPROVED** [review]
Looks great, thanks @gkatsev !

**Pull request merged**